pkg/v1/image.go

```diff
@@ -56,3 +56,8 @@ type Image interface {
 	// (the uncompressed hash).
 	LayerByDiffID(Hash) (Layer, error)
 }
+
+type ImageOpener interface {
```

Comment:
I think if we can make `remote.ImageOpener` private, the value of this diminishes.  IDK that folks will really want to pass around `ImageOpener`s like factories for `Image`s.

Reply:
yep, this actually was meant to be removed regardless of making `ImageOpener` private in various packages.